qa/workunits/test_telemetry_reef_x.sh

```diff
@@ -20,7 +20,7 @@ REPORTED_COLLECTIONS=$(ceph telemetry collection ls)
 NUM_REPORTED_COLLECTIONS=$(echo "$REPORTED_COLLECTIONS" | awk '/^NAME/ {flag=1; next} flag' | wc -l)
 KNOWN_COLLECTIONS=("basic_base" "basic_mds_metadata" "basic_pool_flags" "basic_pool_options_bluestore"
 	           "basic_pool_usage" "basic_rook_v01" "basic_usage_by_class" "crash_base" "device_base"
-		   "ident_base" "perf_memory_metrics" "perf_perf")
+		   "ident_base" "perf_memory_metrics" "perf_perf" "basic_stretch_cluster")
```

Comment:
This workunit is correct to update since it's testing the known collections after upgrading to "X", or main, which now has the collection "basic_stretch_cluster" with this PR.